app/resources/admin.html.template

```diff
@@ -36,6 +36,12 @@
   <input type=submit value="Delete (with confirmation)">
 </form>
 
+<h2>Setup datastore</h2>
+<form method="post" class="admin">
```

Comment:
How about showing this button in all pages including the start page:
https://google.org/personfinder/global/home.html
instead of the real content, if the database is not initialized yet?

If we show this button only in the admin page, it would be hard to notice. All requests go through Main handler in main.py, so I believe you can hook all requests there.